Add meeting agenda 2024-03-13 (#638)
solid/specification: meetings/2024-03-13.md

# W3C Solid Community Group: Weekly

* Date: 2024-03-13T14:00:00Z
* Call: https://meet.jit.si/solid-cg
* Chat: https://matrix.to/#/#solid_specification:gitter.im
* Repository: https://github.com/solid/specification
* Status: Published

## Present
* [Virginia Balseiro](https://virginiabalseiro.com/#me)
* [Sarven Capadisli](https://csarven.ca/#i)
* [Jesse Wright](https://www.jeswr.org/#me)
* Hadrian Zbarcea (Inrupt)
* Jeff Zucker
* [elf Pavlik](http://elf-pavlik.hackers4peace.net)
* Maxime Lecoq
* April Daly
* [Rahul Gupta](https://cxres.pages.dev/profile#i)
* [Matthias Evering](https://solidweb.me/testpro/)

### Regrets

> * Tim BL

Reviewer: ```suggestion
* Tim BL
* [Ted Thibodeau](https://github.com/TallTed/) (he/him) ([OpenLink Software](https://www.openlinksw.com/))
```

* Join queue to talk.
* Topics can be proposed at the bottom of the agenda to be discussed as time allows. Make it known if a topic is urgent or cannot be postponed.

### Participation and Code of Conduct
* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
* [Solid Code of Conduct](https://github.com/solid/process/blob/main/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
* Operating principle for effective participation is to allow access across disabilities, across country borders, and across time. Feedback on tooling and meeting timing is welcome.
* If this is your first time, welcome! please introduce yourself.

### Scribes
* Sarven Capadisli

### Introductions
* name: text

---

## Topics

### Special Topic Meetings
URL: https://github.com/orgs/solid/projects/16/views/1

> * VB: re https://github.com/solid/specification/discussions/555 , Ted mentioned to deprecate so I marked it as archived. Agree that I will move what is discussed. We don't have discussed status anymore on the board. Having done may be enough. Try and see how it goes.

Reviewer: ```suggestion
* VB: re https://github.com/solid/specification/discussions/555 , Ted mentioned to deprecate so I marked it as archived. Agree that I will move what is discussed. We don't have "discussed" status anymore on the board. Having done may be enough. Try and see how it goes.
```

> * eP: We can't put all discussions into done because sometimes we decide to come back to discuss. After the meeting we know we are not "done" and need to come back at osme point. Perhaps a way to mark as in progress?

Reviewer: ```suggestion
* eP: We can't put all discussions into done because sometimes we decide to come back to discuss. After the meeting we know we are not "done" and need to come back at some point. Perhaps a way to mark as in progress?
```

### Meeting recording and publication
URL: https://github.com/w3c-cg/solid/issues/18

* VB: This follows the issue. Please respond in.
* SC: This could automate some things... depends what the group wants.

### Adopt Solid-PREP
URL: https://github.com/solid/specification/issues/580

* RG: I had already proposed Solid-PREP for adoption by the CG. Now Pavlik has added himself as a second author fulfilling the two author requirement. So I would ask the group to adopt it.
* eP: Since we want at least two people working on it, I volunteer. Some text about operations could move to Solid Protocol / Solid Notifications. Worthwhile iterating.
* VB: Any objections to adapt to work item?
* eP: What are the next steps? Needs to add one of the collaborators to transfer the repo?
* SC: I can help with transfer. We can also add it to the homepage (TR).
* eP: re adding to TR/, 
* SC: We'll link to its own repo and ED on the GitHub Pages.

### Initial Best Security Practice
URL: https://github.com/solid/specification/pull/625

* VB: Proposed by eP.

> * eP: Created issue 636 to track some of the related issues. The original issue 514 which was created a year ago. In the whole year we didn't process, document, capture. Sounds reasonable to have a new work item. As long as we have a clear place that people can find re potential security vulnerabilities. In some ways addressing some of the problems. Doesn't have to be perfect but better way than addressing security related issues.

Reviewer: ```suggestion
* eP: Created [issue 636](https://github.com/solid/specification/issues/636) to track some of the related issues. The original [issue 514](https://github.com/solid/specification/issues/514) was created a year ago. In the whole year we didn't process, document, capture. Sounds reasonable to have a new work item. As long as we have a clear place that people can find potential security vulnerabilities. In some ways addressing some of the problems. Doesn't have to be perfect but better way than addressing security related issues.
```

* VB: Work item sounds good.

> * SC: I'm for the idea to more visibility. Suggest to have more care on the language, best practice or guideline? If BP make sure there's sufficient data that signals that. Fine as a work item (best practice and guidelines).

Reviewer: ```suggestion
* SC: I'm for the idea of more visibility. Suggest to have more care on the language, e.g., "best practice" or "guideline". If BP make sure there's sufficient data that signals that. Fine as a work item (best practice and guidelines).
```

* eP: The naming is not most important. Just copied from OAuth Security BP. We need to be clear of course not to compare with stuff like that. Nor particularly comparable now. Early draft. Aspirations to have something like this in the future. We can rename in the future.

> * SC: Title is not the issue but the language that's used in the document. Can discussed later.

Reviewer: ```suggestion
* SC: Title is not the issue, but the language that's used in the document. Can discuss later.
```

* RG: Let's make a work item and wait for it to reach a certain maturity to add to TR.

> * eP: As a rule of thumb: possibly threats and counter measures. If not conflicting with the spec, it is fair / valid approach. At least it is conforming for Solid group of specifications. If measure is passing the Solid test suites, it'd be useful to share it with others.

Reviewer: ```suggestion
* eP: As a rule of thumb: possible threats and countermeasures. If not conflicting with the spec, it is fair/valid approach. At least it is conforming for Solid group of specifications. If measure is passing the Solid test suites, it'd be useful to share it with others.
```

#### Best Security Practice work item proposal
URL: https://github.com/solid/specification/issues/636

* VB: What are the next steps for this? Needs review?

> * eP: Where should it go? It'd good to merge somewhere and then keep iterating. I'm the only one listed at the moment. I want to ask others to collaborate.

Reviewer: ```suggestion
* eP: Where should it go? It'd be good to merge somewhere and then keep iterating. I'm the only one listed at the moment. I want to ask others to collaborate.
```

* VB: Do that and we can discuss next week and in the meantime we can discuss where to put it.

ACTION: eP to ask others to co-authors. The Group revisits the best place for the work to continue.

> * SC: Hard to say what best place is. Keep it in its own repo for now. Security practices cross all specs not just Protocol. If we want to always work on it we can think of it as a live document. I don't see it the same as the Solid Protocol

Reviewer: ```suggestion
* SC: Hard to say what best place is. Keep it in its own repo for now. Security practices cross all specs, not just Protocol. If we want to always work on it, we can think of it as a live document. I don't see it being the same as the Solid Protocol
```

* eP: Also see it better as a dedicated repo for it. Easier to migrate into solid/specification.

ACTION: VB to create repo with `foo-security-enterprises`.

* SC: `solid-security-bp`?
* VB: Will ask in chat.
* eP: `security-topics`
* SC: Nod.
* SC: BP might be more familiarity in/around W3C.
* eP: Follow up in issue.

### Add Contributing Guide PR processing information
URL: https://github.com/w3c-cg/solid/pull/19

> * VB: We already have w3c-cg/solid CONTRIBUTING.md for Contributing for all sorts of stuff. Ther is a section on meeting minutes recently. This PR is specifically about the Contributing Guide itself. I've approved it. It has been open for about a week? If you want to make a suggestion to the Contributing, this sets some expectations on processing.

Reviewer: ```suggestion
* VB: We already have w3c-cg/solid CONTRIBUTING.md for contributing all sorts of stuff. There is a section on meeting minutes recently. This PR is specifically about the Contributing Guide itself. I've approved it. It has been open for about a week? If you want to make a suggestion to the Contributing, this sets some expectations on processing.
```

* eP: Right, if no objections, lets merge. I've approved.
* VB: Merged.

> * SC: Minor comment: we have contributing guide because it's a rough approximation of what seems to work in the group. We do't always hit deadlines or follow to the dot, but the more wwe take it seriously we speed things up and there's less chatter on how to work. Not about watching over each other's shoulder. 

Reviewer: ```suggestion
* SC: Minor comment: we have contributing guide because it's a rough approximation of what seems to work in the group. We do't always hit deadlines or follow to the dot, but the more wwe take it seriously, the more we speed things up and the less chatter on how to work. Not about watching over each other's shoulder. 
```

* VB: For me it is important also to follow guidelines in particular regarding the minutes. I get messages about it from folks. So, good to have the minutes out.
* SC: (I've paraphrased a bit above.)

### Update on ecosystem tests work
URL: https://github.com/solid-contrib/ecosystem-tests

* MdJ: We're starting ecosystem tests with headless browsers. Making progress on switch from launcher-exploration to SAI.
* VB: Anyone would like to talk about these topics?

### ACP Tooling
URL: https://www.inrupt.com/blog/podbrowser-sunset

* MdJ: Does podbrowser sunset create a gap that needs filling?
* VB: Postpone until MdJ is here? Okay.

### SPICE: Secure Patterns for Internet CrEdentials BoF at IETF

* RG: For authorization related stuff : https://datatracker.ietf.org/meeting/119/agenda
Tuesday: Session 2 

### Solid March Mini-Hackathon
URL: https://solidhack.org/

> * eP: Mostly Jackson. Hadrian, MdJ and I are helping. mostly for shapes that people are working with. We're inviting people. I think there'd be more focus. There is a dedicated Matrix channel. If someone wants to work with shapes, especially ShEx, everyone is welcome.

Reviewer: ```suggestion
* eP: Mostly Jackson. Hadrian, MdJ, and I are helping. Mostly for shapes that people are working with. We're inviting people. I think there'd be more focus. There is a dedicated Matrix channel. If someone wants to work with shapes, especially ShEx, everyone is welcome.
```

Reviewer: Would be good to also add a link to the Matrix channel...

ACTION: eP to ask others to co-authors. The Group revisits the best place for the work to continue.

* SC: Hard to say what best place is. Keep it in its own repo for now. Security practices cross all specs, not just Protocol. If we want to always work on it, we can think of it as a live document. I don't see it being the same as the Solid Protocol
* eP: Also see it better as a dedicated repo for it. Easier to migrate into solid/specification.

ACTION: VB to create repo with `foo-security-enterprises`.

* SC: `solid-security-bp`?
* VB: Will ask in chat.
* eP: `security-topics`
* SC: Nod.
* SC: BP might be more familiarity in/around W3C.
* eP: Follow up in issue.

### Add Contributing Guide PR processing information
URL: https://github.com/w3c-cg/solid/pull/19

* VB: We already have w3c-cg/solid CONTRIBUTING.md for contributing all sorts of stuff. There is a section on meeting minutes recently. This PR is specifically about the Contributing Guide itself. I've approved it. It has been open for about a week? If you want to make a suggestion to the Contributing, this sets some expectations on processing.
* eP: Right, if no objections, lets merge. I've approved.
* VB: Merged.

> * SC: Minor comment: we have contributing guide because it's a rough approximation of what seems to work in the group. We do't always hit deadlines or follow to the dot, but the more wwe take it seriously, the more we speed things up and the less chatter on how to work. Not about watching over each other's shoulder. 

Reviewer: ```suggestion
* SC: Minor comment: we have contributing guide because it's a rough approximation of what seems to work in the group. We don't always hit deadlines or follow everything to the dot, but the more we succeed (or at least take it seriously), the more we speed things up and the less chatter on how to work. Not about watching over each other's shoulder. As mentioned before, it helps us to be more efficient and be on the same page, as well as being useful to newcomers and orientation.
```

# W3C Solid Community Group: Weekly

* Date: 2024-03-13T14:00:00Z
* Call: https://meet.jit.si/solid-cg
* Chat: https://matrix.to/#/#solid_specification:gitter.im
* Repository: https://github.com/solid/specification
* Status: Draft

## Present
* [Virginia Balseiro](https://virginiabalseiro.com/#me)

> 

Reviewer 2: Add: regrets from Tim BL
